contrib/src/main/java/com/datatorrent/contrib/schema/formatter/Formatter.java

```diff
@@ -0,0 +1,53 @@
+package com.datatorrent.contrib.schema.formatter;
+
+import com.datatorrent.api.Context;
+import com.datatorrent.api.Context.PortContext;
+import com.datatorrent.api.DefaultInputPort;
+import com.datatorrent.api.DefaultOutputPort;
+import com.datatorrent.api.Operator.ActivationListener;
+import com.datatorrent.api.annotation.InputPortFieldAnnotation;
+import com.datatorrent.api.annotation.OutputPortFieldAnnotation;
+import com.datatorrent.common.util.BaseOperator;
+
+public abstract class Formatter <INPUT, OUTPUT> extends BaseOperator implements ActivationListener<Context>
```

Comment:
Can you make Formatter implements Converter? so that it can be used as standalone converter in other places.